packages/docs/docs/developerGuide/developerGuide.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
how about using `index.md` as file name, when appropriate, in the same way as we do for JS?

Reply:
I think we should not use index.md but call the files the same as the id we use. However likely setup/setup is not ideal naming

Reply:
but why? :-)

Reply:
ok I don't know what "id" is in this case